spec/lib/abstractivator/tree_visitor_spec.rb

```diff
@@ -171,6 +173,10 @@ def self.example(description, values)
       end
     end
 
+    def self.build_set(count)
```

Comment:
This helps DRY the specs, but for me, it actually makes the tests less readable. I would take the tradeoff and put the literal structures right under the `example`. That fits with the existing test style better, too.